src/components/dialog/DialogDismissibleView.tsx

```diff
@@ -54,216 +55,200 @@ interface LocationProps {
 }
 
 const DEFAULT_DIRECTION = PanningProvider.Directions.DOWN;
+const TOP_INSET = Constants.isIphoneX ? Constants.getSafeAreaInsets().top : Constants.isIOS ? 20 : 0;
+const BOTTOM_INSET = Constants.isIphoneX ? Constants.getSafeAreaInsets().bottom : Constants.isIOS ? 20 : 0;
+
+const DialogDismissibleView = (props: Props) => {
+  const {direction = DEFAULT_DIRECTION, visible: propsVisible, containerStyle, style, children, onDismiss} = props;
+  // @ts-expect-error
+  const {isPanning, dragDeltas, swipeDirections} = useContext<PanContextProps>(PanningContext);
+
+  const width = useRef<number>(Constants.screenWidth);
+  const height = useRef<number>(Constants.screenHeight);
+  const thresholdX = useRef<number>(0);
+  const thresholdY = useRef<number>(0);
+  const counter = useRef<number>(0);
+  const containerRef = useRef<typeof View>();
+  const animatedValue = useRef<Animated.AnimatedValue>(new Animated.Value(0));
+  const swipe = useRef<PanDirectionsProps>({});
```

Comment:
Same here with `swipe`, consider renaming to something more readable

Reply:
Changed to mutableSwipeDirections